docs/DevGuide.md

```diff
@@ -67,9 +63,7 @@ When running locally:
                 "applicationUrl": "https://localhost:53180",
                 "environmentVariables": {
                     "ASPNETCORE_ENVIRONMENT": "Development",
-                    "VmrPath": "D:\\tmp\\dnceng-vmr",
-                    "TmpPath": "D:\\tmp\\",
-                    "VmrUri": "https://github.com/maestro-auth-test/dnceng-vmr"
+                    "TmpPath": "D:\\tmp\\"
```

Comment:
We might be overusing the `TmpPath` folder a bit. It's probably not for this PR, but maybe we should have a different path for storing the VMRs, since they're not very temporary, or maybe just rename this into something else, like `RepositoriesPath`, since Tmp sounds like they get deleted at some point

Reply:
I think the repos and the VMRs are on the same level really - you can stop the service, delete the temp and run it again and everything will work. You can even delete these between work items and everything will still work.

Also the VMRs get cloned under `TmPath/vmrs/` so they are in a different spot.

Reply:
I guess from that standpoint, they are temporary, it's just not something you probably want to do since it takes a lot of time

Reply:
let's keep it as is then